x/incentives/client/cli/tx.go

```diff
@@ -33,7 +33,7 @@ func NewCreateGaugeCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create-gauge [lockup_denom] [reward] [poolId] [flags]",
 		Short: "create a gauge to distribute rewards to users",
```

Comment:
can we adda description here that states "for duration lock gauges set poolId = 0 and for all CL (no-lock) gauges set it to a CL poolId"?